packages/docs/docs/install.mdx

```diff
@@ -4,10 +4,10 @@ title: Install
 
 # Install
 
-Before using our SDK, you first need to install the Fuel wallet browser extension.
-To install, download the [FuelWallet zip file](https://wallet.fuel.network/app/fuel-wallet.zip) and import directly on your extensions management page on Chrome or Brave.
-You can also watch the video below to see how to install.
-Since the wallet is still in an alpha version, it isn't possible to download directly from the Chrome Web Store.
+Before starting using our SDK, you need first to install Fuel wallet browser extension.
```

Comment:
```suggestion
Before using our SDK, you need first to install Fuel wallet browser extension.
```
or "Before you start using our SDK..."

Reply:
"Before using our SDK, you need to install Fuel wallet browser extension" is more concise.

Reply:
"Before using our SDK, you need to install the Fuel Wallet browser extension" would be better imo. If you want to take out the "the" it would have to be referring to a name like "you need to install Fuel Wallet".